apps/checkout-app/backend/middlewares.ts

```diff
@@ -3,15 +3,18 @@ import { Response } from "retes/response";
 import { withSaleorDomainPresent } from "@saleor/app-sdk/middleware";
 import { SALEOR_DOMAIN_HEADER } from "@saleor/app-sdk/const";
 import { getSaleorDomain } from "./utils";
+import { getErrorMessage } from "@/checkout-app/utils/errors";
 
 export const withSaleorDomainMatch: Middleware = (handler) =>
   withSaleorDomainPresent((request) => {
-    const saleorDomain = getSaleorDomain();
+    let saleorDomain;
```

Comment:
@mmiszy this looks like implicit `any`, I don't know why TS even allows it.

Maybe we can add this case to ESLint to prevent this globally?

![Zrzut ekranu 2022-07-15 o 12 11 27](https://user-images.githubusercontent.com/9268745/179203038-3badce0c-f454-4277-9c03-8d7d8f8944ae.png)
t